test/testwindow.go

```diff
@@ -68,6 +69,13 @@ func (w *testWindow) SetOnClosed(closed func()) {
 
 func (w *testWindow) Show() {}
 
+func (w *testWindow) Clipboard() fyne.Clipboard {
+	if w.clipboard == nil {
+		w.clipboard = &testClipboard{}
```

Comment:
Just set this in the constructor - much simpler as the testClipboard is tiny.
Avoids possible race conditions.